mantle/kola/harness.go

```diff
@@ -335,6 +340,17 @@ func parseDenyListYaml() error {
 			continue
 		}
 
+		if obj.SnoozeDate != "" {
+			snoozeDate, err := time.Parse(snoozeFormat, obj.SnoozeDate)
+			if err != nil {
+				return err
+			} else if today.After(snoozeDate) {
+				continue
+			}
+
+			fmt.Printf("🕒 Snoozing test pattern \"%s\" until %s\n", obj.Pattern, snoozeDate.Format("Jan 02 2006"))
```

Comment:
Maybe

```suggestion
			fmt.Printf("🕒 Snoozing kola test pattern \"%s\" until %s\n", obj.Pattern, snoozeDate.Format("Jan 02 2006"))
```

to be consistent with the wording of the other output?